contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -17,11 +17,11 @@ GRANT SELECT on sys.babelfish_sysdatabases TO PUBLIC;
 -- BABELFISH_SCHEMA_PERMISSIONS
 CREATE TABLE sys.babelfish_schema_permissions (
   dbid smallint NOT NULL,
-  schema_name NAME NOT NULL,
-  object_name NAME NOT NULL,
-  permission NAME NOT NULL,
-  grantee NAME NOT NULL,
-  object_type NAME,
+  schema_name NAME NOT NULL COLLATE sys.database_default,
+  object_name NAME NOT NULL COLLATE sys.database_default,
+  permission NAME NOT NULL COLLATE sys.database_default,
+  grantee NAME NOT NULL COLLATE sys.database_default,
+  object_type NAME COLLATE sys.database_default,
   PRIMARY KEY(dbid, schema_name, object_name, permission, grantee)
 );
```

Comment:
Sorry just one more comment - 
I understand there may be some security concern so we are not exposing this catalog to user, but I think we should still select from this table during JDBC test to make sure all the entries are updated as expected, especially during upgrade. 
Would you please add that into the test? We can do this through psql. Otherwise code lgtm.